src/portable/portable.c

```diff
@@ -556,14 +571,14 @@ int portable_extract(
                 }
         }
 
-        r = portable_extract_by_path(image->path, true, matches, &os_release, &unit_files, error);
+        r = portable_extract_by_path(image->path, false, matches, &os_release, &unit_files, error);
```

Comment:
maybe add `/*  path_is_extension= */` comment before the false? just to make clearer what boolena is passed there